vcpkg.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "minetest",
+	"version-string": "5.5.0",
+	"dependencies": [
+		"zlib",
+		"zstd",
+		{
+			"name": "curl",
+			"features": [
+				"winssl"
+			]
+		},
+		"openal-soft",
+		"libvorbis",
+		"libogg",
+		"libjpeg-turbo",
+		"sqlite3",
+		"freetype",
+		"luajit",
+		"gmp",
+		"jsoncpp",
+		"opengl-registry"
+	]
```

Comment:
...then this file can also be moved to `util/ci/`, the action has an `vcpkgJsonGlob` argument that allows specifying the path.